Xamarin.Essentials/TextToSpeech/TextToSpeech.mac.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using AppKit;
+
+namespace Xamarin.Essentials
+{
+    public static partial class TextToSpeech
+    {
+        static readonly Lazy<NSSpeechSynthesizer> speechSynthesizer = new Lazy<NSSpeechSynthesizer>(() =>
+            new NSSpeechSynthesizer { Delegate = new SpeechSynthesizerDelegate() });
+
+        internal static Task<IEnumerable<Locale>> PlatformGetLocalesAsync() =>
+            Task.FromResult(NSSpeechSynthesizer.AvailableVoices
+                .Select(v => new Locale(v, null, null, null)));
+
+        internal static async Task PlatformSpeakAsync(string text, SpeechOptions options, CancellationToken cancelToken = default)
+        {
+            var ss = speechSynthesizer.Value;
+            var ssd = (SpeechSynthesizerDelegate)ss.Delegate;
+
+            var tcs = new TaskCompletionSource<bool>();
+            try
+            {
+                // Ensures linker doesn't remove.
```

Comment:
Story on this bit?

Reply:
Lazy is too lazy
https://github.com/xamarin/Essentials/pull/1209#discussion_r405051562